Automattic/wp-calypso
# Adding Cloudflare cards.

This PR adds Cloudflare cards to:
* The Stats page.
* The primary section of My Home (companion WPcom patch D55161-code)
* And the Settings > Performance page

Remaining to-dos:
* Plug in actual links
* Finalize illustrations

#### Testing instructions
* Checkout this PR
* Apply D55161 to your sandbox
* Sandbox WPcom api calls
* Navigate to My Home and cycle through the Task cards until you see and click the link
* Navigate to the Stats page and click the link
* Navigate to Settings > Performance and click the link
* Verify copy and layout for all cards

Fixes #
